addons/tmf_loadouts/loadouts/ru_army_template_2020.hpp

```diff
@@ -555,13 +555,15 @@ class sp : g {
 	};
 };
 
-class vg : smg {
+class vg : car {
 	displayName = "Vehicle Gunner";
 	goggles[] = {"default"};
-	vest[] = {"CUP_V_RUS_6B45_1"};
-	headgear[] = {"H_Tank_black_F"};
+	vest[] = {"CFP_V_O_RUMVD_SURPAT"};
+	headgear[] = {"H_HelmetCrew_I"};
+	sidearmWeapon[] = {"hgun_Rook40_F"};
```

Comment:
No point in assigning both handguns and primaries to everyone. Maybe if you were to only assign a primary to the commander then it would make some sense.

Reply:
Ooh, good point. Might give the commander the long arm and give the crew sidearms then.